frameworks/elastic/tests/test_shakedown.py

```diff
@@ -151,7 +150,28 @@ def test_master_node_replace():
     # Ideally, the pod will get placed on a different agent. This test will verify that the remaining two masters
     # find the replaced master at its new IP address. This requires a reasonably low TTL for Java DNS lookups.
     cmd.run_cli('elastic --name={} pod replace master-0'.format(FOLDERED_SERVICE_NAME))
-    # setup_function will verify that the cluster becomes healthy again.
+    # pre_test_setup will verify that the cluster becomes healthy again.
```

Comment:
Would it not make more sense to have a post-test teardown method performing the checks? In this way, the test that caused the failure will show up as a failure in the test logs. Is this possible with pytest?
(This may be out of scope of this PR)